services/blockchain-indexer/shared/indexer/accountIndex.js

```diff
@@ -108,21 +107,6 @@ const triggerAccountUpdates = async () => {
 
 const indexAccountWithData = (account) => accountDirectUpdateQueue.add(account);
 
-const indexAllDelegateAccounts = async () => {
-	const accountsDB = await getAccountIndex();
-	const allDelegatesInfo = await requestRpc('invokeAction', { action: 'dpos:getAllDelegates', params: {} });
-	const allDelegateAddresses = allDelegatesInfo.map(({ address }) => address);
-	const PAGE_SIZE = 1000;
-	for (let i = 0; i < Math.ceil(allDelegateAddresses.length / PAGE_SIZE); i++) {
-		/* eslint-disable no-await-in-loop */
-		const accounts = await getAccountsByAddress(allDelegateAddresses
-			.slice(i * PAGE_SIZE, (i + 1) * PAGE_SIZE));
-		await accountsDB.upsert(accounts);
-		/* eslint-enable no-await-in-loop */
-	}
-	logger.info(`Indexed ${allDelegateAddresses.length} delegate accounts`);
-};
-
 const buildLegacyAccountCache = async () => {
 	// Cache the legacy account reclaim balance information
 	const [genesisBlock] = await getBlockByHeight(await getGenesisHeight(), true);
```

Comment:
This could make use of the `getGenesisAccounts` method. Let us discuss this on the call.